requirements/base.txt

```diff
@@ -40,4 +40,4 @@ Markdown==2.6.8
 # Note: also adjust pootle/checks::TTK_MINIMUM_REQUIRED_VERSION
 translate-toolkit==2.1.0
 # If you want to use Translate Toolkit 'master'
-#-e git://github.com/translate/translate.git#egg=translate-toolkit-2.0.0
+#-e git+https://github.com/translate/translate.git#egg=translate-toolkit-2.2.0
```

Comment:
Please add the why to the commit message, it isn't required, it's only required for older pip versions.  Cause git: is faster we probably want to bring that back at some time.